polygon/websocket/__init__.py

```diff
@@ -5,6 +5,7 @@
 import inspect
 import ssl
 import certifi
+from multiprocessing import AuthenticationError
```

Comment:
Make our own instead of importing from `multiprocessing` (whatever that is)

Reply:
Should I just add the new class at the top of the file or should it go somewhere else?

Reply:
New class at the top is fine.